### DISCUSSION TITLE
still using your fix on 22H2

### DISCUSSION THREAD

**Discussion by abigor82** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#94] [General]
i did fresh install windows 11 22H2(10.0.22621.1413)  with latest updates... drag and drop to the task still acts out or don't work at all.
i had to run your app get my drag and drop work properly. im a heavy user of this feature im this close going back to win10...

thank you making this fix...

**Comment by HerMajestyDrMona** [comment]
Yes, unfortunately some users have already reported that the drag and drop still isn't working for them in Win11 22H2. Did you by any chance turn off the UAC using registry modification? https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/discussions/93

**Reply by abigor82** [reply]
Hi, so im using ghost Spectre   windows 11 22H2 iso which enable  main administrator account  after install. I read few forums soneone did mention if uac is disable may cause drag and drop stop working so. I did registry edit enable the uac  still didn't work.

**Reply by HerMajestyDrMona** [reply]
Oh well. It's still some modified version of Windows 11 after all :(

**Reply by abigor82** [reply]
Well I don't mine using ur app in the background i got ton of memory on my gaming rig.  Microsoft, can kiss my butt. This was not issue with win10.

**Reply by HerMajestyDrMona** [reply]
I'm glad that the program still works for you, however it might stop working one day, and I will most likely not update it. The native drag-and-drop support probably works much better than the workaround program with unfixable limitations (such as Virtual machines windows). 
I wonder if there's maybe some [ViVeTool](https://github.com/thebookisclosed/ViVe) code that would enable drag-and-drop in your Windows 11 build?

**Reply by abigor82** [reply]
hi... so i figured out the problem was... since have superlite version of Windows 11 22H2 i had dig in the toolbox  program comes with the OS
enable the taskbar drag and drop is odd they disable it 
here screenshot...
https://i.postimg.cc/4xPPjmS3/untitled.png
hopefully this will help others if you have de-bloated or modded version of windows 11

**Reply by Perplexer1** [reply]
I also noticed that disabling UAC via the registry by setting EnableLUA to "0" under "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System" stops taskbar drag & drop from working. Another user mentioned that the same happens when his computer joins a domain. So Microsoft need to do some more work.

**Comment by LiquidatorFX** [comment]
I don’t know what they fixed there with these jacks from Microsoft, but without this stray nothing works on the newest builds the same way it didn’t work before. Maybe this needs to be enabled separately somewhere in the HP registries, but the fact is that just as it didn’t work, it still doesn’t work normally without your program

**Comment by LiquidatorFX** [comment]
and after the latest Windows 11 update, it became difficult to get to an already open tab when dragging a file there, that is, I move the cursor to the tab, and a tab opens 6-7 icons to the right of where I hovered the mouse.... this is fucked up

**Comment by LiquidatorFX** [comment]
Is it possible to get an update for your program so that it works properly? Or how can you enable this using Windows itself? Nothing works for me from the built-in administrator account - only your program but shifting the open tab to the right relative to the one to which I moved the cursor
